### PR TITLE
Connectors Alpha - Store updates

### PR DESCRIPTION
**Pull request by danielsousaio**
This PR hooks the new redux files inside `src/store` to the app.

### CODE REVIEW

File: {{cookiecutter.project_slug}}/src/store/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { apiReducer } from "./reducers";
+import { combineReducers } from "redux";
+import { createStore, applyMiddleware, compose } from "redux";
```
Comment (tan-duong) [inline {{cookiecutter.project_slug}}/src/store/index.js:3]: combineReducers should be in one import line e.g. line 3

File: {{cookiecutter.project_slug}}/App.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import {ApplicationProvider, Layout, Text, IconRegistry} from 'react-native-ui-k
 import {Provider as ReduxProvider} from 'react-redux';
 
 import SplashScreen from './src/features/SplashScreen';
-import {store} from './src/redux/store';
+import {store} from './src/store';
```
Comment (tan-duong) [inline {{cookiecutter.project_slug}}/App.js:7]: this breaks all other redux states in current modules

File: {{cookiecutter.project_slug}}/src/store/index.js (hunk 1 above)
Comment (danielsousaio) [reply {{cookiecutter.project_slug}}/src/store/index.js]: Fixed in cab7061

**Comment by tan-duong** [comment]
as discussed via the call, let's get this moved

**Pull request merged**